lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -3460,6 +3465,37 @@ SPIRVValue *LLVMToSPIRVBase::transIntrinsicInst(IntrinsicInst *II,
   return nullptr;
 }
 
+SPIRVValue *LLVMToSPIRVBase::transFenceInst(FenceInst *FI,
+                                            SPIRVBasicBlock *BB) {
+  SPIRVWord MemorySemantics = static_cast<SPIRVWord>(FI->getOrdering());
+  // Fence ordering may only be Acquire, Release, AcquireRelease, or
+  // SequentiallyConsistent
+  switch (MemorySemantics) {
+  case 4:
```

Comment:
Guess, Ordering starts with '3', assuming NotAtomic is '0' (at least https://llvm.org/doxygen/namespacellvm.html#a9bccbe67aaab722783ca4e7c504aaaa7 makes me thinking so)

Reply:
https://llvm.org/doxygen/AtomicOrdering_8h_source.html#l00056 I guess 3 should be Consume and it is "Not specified yet"